lib/internal/Magento/Framework/Cache/Frontend/Decorator/Logger.php

```diff
@@ -44,7 +44,7 @@ public function remove($identifier)
      */
     public function clean($mode = \Zend_Cache::CLEANING_MODE_ALL, array $tags = [])
     {
-        $result = parent::clean($mode, $tags, $mode);
+        $result = parent::clean($mode, $tags);
```

Comment:
public function clean($mode = \Zend_Cache::CLEANING_MODE_ALL, array $tags = [])
    {
        return $this->_getFrontend()->clean($mode, $tags);
    }